src/common/include/SIRF/common/mr_image_data.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "sirf_image_data.h"
+
+namespace sirf {
+	template <typename T>
+	class MRImageData : public SIRFImageData<T>
+	{
+	public:
+		virtual ~MRImageData() {}
```

Comment:
not necessary. we need a virtual destructor in the base class, therefore none in the derived classes (unless it needs to do something special)